cdn/webpack.config.js

```diff
@@ -163,8 +186,46 @@ const config = (target) => {
           }
         }
       ]
-    }
+    },
+    output: {
+      ...commonConfig.output,
```

Comment:
[suggestion] Use [webpack-merge](https://www.npmjs.com/package/webpack-merge) and you don't have to worry about doing stuff like this.